desktop/bin/make-changelog.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+CALYPSO_DIR=$(cd $(dirname $0)/../../ && pwd)
+
+# desktop tags
+function get_desktop_tags() {
+  desktop_tags=""
+  declare -i x=0
+  for tag in $(git tag); do
+    # guard against stray v9.9.9 test tag that keeps being re-introduced to the repo! ಠ_ಠ
```

Comment:
Maybe some people are `git push --tags` instead of `git push origin <tag>`?

Reply:
Yeah - perhaps. Although Calypso isn't tagged that frequently so not sure how this keeps happening. I should probably PSA this to let people know to delete this tag locally if they have it - hopefully this isn't the result of some automation somewhere. 😆